src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -296,6 +296,9 @@ class FuzzingStrategies(object):
   # Probability for transforming CMPLOG solving.
   CMPLOG_TRANS_PROB = 0.1
 
+  # Probability for randomized coloring.
```

Comment:
What does coloring mean in this context?

Reply:
cmplog/redqueen has two phases, first the coloring phase, then the insertion phase. in the coloring phase it checks where in the input bytes change the outcome of the coverage and where not. currently this is done with a semi-random map, where an upcase letter is switched with a different upcase letter.
a user reported that this is not good in his fuzzing case, and totally random switching works better. so he added the option in a PR to fully randomize it, so that an upcase letter can also be a number or 0x02.
fuzzbench showed that this is normally much worse than what I implemented, but in a few cases this can be good, hence activating this with a 10% chance.

Reply:
Gotcha.